middleware/index.js

```diff
@@ -81,4 +81,4 @@ class GutenbergMiddleWare {
 	}
 }
 
-window.gutenbergMiddleWare = new GutenbergMiddleWare();
+window.gutenbergFieldsMiddleWare = new gutenbergFieldsMiddleWare();
```

Comment:
@danielbachhuber I think it should be `new GutenbergFieldsMiddleWare()` ?

Reply:
Updated in de0c036